web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/page/ActionsList/DetailsMenu/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { EditIcon } from "@/components/Icons/EditIcon";
+import { ElementsIcon } from "@/components/Icons/ElementsIcon";
+import { Menu } from "@headlessui/react";
+import Link from "next/link";
+
+export const DetailsMenu = (props: { path: string }) => {
+  const { path } = props;
+  return (
+    <Menu as="div" className="relative inline-block">
+      <Menu.Button className="w-8 h-8 flex justify-center items-center p-2 hover:bg-grey-100 rounded-lg">
+        <ElementsIcon />
+      </Menu.Button>
+      <Menu.Items className="transform absolute right-0 shadow-button mt-1 py-2 pr-10 pl-2 origin-top-right bg-white hover:bg-grey-50  border-grey-100 border rounded-lg  ring-0 focus:outline-none">
```

Comment:
@andy-t-wang we should add some z-index to this element it got cut on the last element of the table.